arches/app/views/search.py

```diff
@@ -178,8 +179,15 @@ def search_terms(request):
             Match(field="displayname.value", query=searchString, fuzziness=2, boost=2)
         )
 
-        if user_is_reviewer is False and index == "terms":
-            boolquery.filter(Terms(field="provisional", terms=["false"]))
+        if index == "terms":
+            boolquery.filter(
+                Terms(
+                    field="nodegroupid", terms=[str(ng) for ng in permitted_nodegroups]
```

Comment:
Even 1000 nodegroups shouldn't be a burden on the limit, looks like the term limit is 65536 by default 👍

Reply:
Perhaps this should interrogate which permission backend is in use to pick an allow vs a deny list?